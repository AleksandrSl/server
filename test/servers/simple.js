#!/usr/bin/env node

var Server = require('../../server')
var app = new Server({
  uniqName: 'server',
  subprotocol: [1, 0],
  supports: [1]
})

app.listen()
