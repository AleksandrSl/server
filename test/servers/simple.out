
 INFO  1970-01-01 00:00:00 Logux server is listening
       Logux Server: 0.0.0
       Server name:  server
       Environment:  test
       Subprotocol:  1.0
       Supports:     1.x
       Listen:       ws://127.0.0.1:1337
